IdentityCore/src/webview/embeddedWebview/ui/mac/MSIDCredentialCollectionController.m

```diff
@@ -0,0 +1,104 @@
+// Copyright (c) Microsoft Corporation.
+// All rights reserved.
+//
+// This code is licensed under the MIT License.
+//
+// Permission is hereby granted, free of charge, to any person obtaining a copy
+// of this software and associated documentation files(the "Software"), to deal
+// in the Software without restriction, including without limitation the rights
+// to use, copy, modify, merge, publish, distribute, sublicense, and / or sell
+// copies of the Software, and to permit persons to whom the Software is
+// furnished to do so, subject to the following conditions :
+//
+// The above copyright notice and this permission notice shall be included in
+// all copies or substantial portions of the Software.
+//
+// THE SOFTWARE IS PROVIDED "AS IS", WITHOUT WARRANTY OF ANY KIND, EXPRESS OR
+// IMPLIED, INCLUDING BUT NOT LIMITED TO THE WARRANTIES OF MERCHANTABILITY,
+// FITNESS FOR A PARTICULAR PURPOSE AND NONINFRINGEMENT. IN NO EVENT SHALL THE
+// AUTHORS OR COPYRIGHT HOLDERS BE LIABLE FOR ANY CLAIM, DAMAGES OR OTHER
+// LIABILITY, WHETHER IN AN ACTION OF CONTRACT, TORT OR OTHERWISE, ARISING FROM,
+// OUT OF OR IN CONNECTION WITH THE SOFTWARE OR THE USE OR OTHER DEALINGS IN
+// THE SOFTWARE.
+
+#import "MSIDCredentialCollectionController.h"
+
+// UI position values for the input dialog
+const CGFloat USERNAME_LABEL_X = 7;
+const CGFloat USERNAME_LABEL_Y = 36;
```

Comment:
nit: consider using some UI component instead of hardcoding the X/Y coordinates. e.g. NSStackView?
Or can't NSAlert just show 2 textfields in it itself?

Reply:
Created an issue for it. Probably we could figure out a better implementation for that later :)
https://github.com/AzureAD/microsoft-authentication-library-common-for-objc/issues/142